chapters/3-measurement.tex

```diff
@@ -37,125 +39,210 @@
+one \textit{data linkage table},\index{Data linkage table}
+one or several \textit{master datasets}\index{Master datasets}
+and one or several \textit{data flow charts}.\index{Data flowchart} 
+
+A \textbf{data linkage table}\sidenote{
+	\url{https://dimewiki.worldbank.org/Data\_Linkage\_Table}}
+lists all the datasets that will be used in the project.
+Its most important function is to indicate 
+how all those datasets can be be linked when
+combining information from multiple data sources.
+\textbf{Master datasets}\sidenote{
 	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}}
-\index{master datasets}
-details all project-wide time-invariant information
-about all observations encountered,
-as well as their relationship to the research design,
-typically summarized by sampling and treatment status.
-Having a plan for how to get the raw data into analysis shape 
-before you acquire it,
-and making sure that the full research team knows where to find this information,
-will save you a ton of time during the course of the project
-and increase the quality of your research.
-
-You should create a master dataset
-for each unit of observation
-relevant to the research.
-This includes all units used for significant research activity,
-like data collection or data analysis.
-Therefore, any unit
-that will be used in sampling or treatment assignment,
-must have a master dataset,
-and that master dataset -- not field data --
-should be used when sampling or assigning treatment.
-Master data sets are often created from field data,
-but master data sets should be treated differently
-as there can be several field datasets,
-but only one authoritative master dataset.
-Master data sets should be created as soon as you
-start to get information about units.
-For example, when receiving a type of administrative data set for the first time,
-or after doing a respondent listing before a survey.
-
-You also need to record how all datasets for each unit of observation
-will link or merge with each other as needed.
-This linking scheme is called a \textbf{data map}\sidenote{
-	\url{https://dimewiki.worldbank.org/data\_map} (TO BE CREATED)}.
-\index{data maps} 
-A data map is more than just a list of datasets.
-Its purpose is to specify the characteristics and linkages of those datasets.
-To link properly, the master datasets must include fully unique ID variables.\sidenote{
-	\url{https://dimewiki.worldbank.org/ID\_Variable\_Properties}}
-The master datasets should indicate whether datasets should be merge one-to-one,
-for example, merging baseline data and endline data that use the same unit of observation,
+list all observations your project ever encounter
+are the authoritative source for all research data
+such as ID values, sampling and treatment status, etc.
+\textbf{Data flow charts}\sidenote{
+	\url{https://dimewiki.worldbank.org/Data\_Flow\_Chart}}
+list all datasets that are needed to create each analysis dataset,
```

Comment:
Some descriptor for the first use of "datasets" in this line? "Raw" or "original"?

Reply:
Later it is explained that each starting point is either a dataset listed in the data linkage table or a master dataset. so it is not just "raw" data sets. I would be ok with changing the text here to: "_list all data sources that are needed to create each analysis dataset,_"

Reply:
done in 173d1d6657ea8a3cd4e79c3a1bd1d55b915b7cbc